CHANGELOG.rst

```diff
@@ -16,6 +16,9 @@ and this project adheres to
 Added
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+- When failover mode is stateful, all manual leader promotions will be consistent:
+  every instance before becoming writable performs `wait_lsn` operation to
+  sync with previous one.
```

Comment:
When the failover mode is `stateful`, all manual leader promotions are consistent: before becoming writable, each instance performs the `wait_lsn` operation to sync up with the previous one.